### ISSUE TITLE
cannot Test Launch a workspace type

### ISSUE THREAD

**Issue by zaro0508** [awslabs/service-workbench-on-aws#302]
I have installed a fresh SC workbench. I can login and browse around. I can add an AWS account, setup an index, setup a project, associate it with an index and assign a project admin user to it. I "import" the EC2 Linux-v1 product into "workspace types" and now I'm attempting to [Test Launch the workspace type](https://pasteboard.co/JO36IcR.png).   On the test launch page the [configuration field is not editable](https://pasteboard.co/JO34ihX.png).  When I click on the "Create Workspace" button it fails saying that the [configuration field is required](https://pasteboard.co/JO34scY.png) but I cannot set that field to anything.  Am I missing something?

➜ node --version
v15.5.1

➜ pnpm --version
5.17.1

➜ go version
go version go1.12.9 darwin/amd64

**Comment by SanketD92** [comment]
Hello, it seems like you've missed adding a configuration for the EC2-linux workspace-type. In order to add one, click on "Edit" on the workspace-type card, then add a configuration under its "Configurations" tab. 

This would need you to enter the instance type, AMI ID and among other settings (You could also use the placeholder values, like `${<placeholder>}` wherever allowed for dynamic selection during provisioning, eg. `${cidr}` for AccessFromCidrBlock).

**Comment by zaro0508** [comment]
thanks @SanketD92 Actually i did add the configurations but apparently I must have made a mistake on one or a few of the configs.  No errors, the workbench just accepted the values i entered so I didn't know.  

I believe I have fixed the values and now I am now able to select a workspace type on the create workspace page.  However I'm now seeing a permissions problem when I create the workspace.  User: arn:aws:sts::XXXXXXXX36:assumed-role/dev-va-sw-WorkflowLoopRunner/va-sw-backend-dev-workflowLoopRunner is not authorized to perform: sts:AssumeRole on resource: arn:aws:iam::aws:policy/AWSServiceCatalogEndUserFullAccess

**Comment by nguyen102** [comment]
Hi, 
1. To confirm, what configuration did you set for the EC2-Linux? Were you able to use the configurations outlined here?
https://github.com/awslabs/service-workbench-on-aws/blob/master/docs/docs/deployment/post_deployment/import_service_catalog_products.md

This file is best viewed when running docusaurus inside of the docs folder. To run docusaurus, go to the docs folder and run `yarn start`

2. Just to confirm, did you set up a member account and assigned an index and project to that account? Instructions [here](https://github.com/awslabs/service-workbench-on-aws/blob/master/docs/docs/deployment/post_deployment/account_structure.md)

3. Let's double check that your workflowLoopRunner IAM role has the correct permission. Go to your parent AWS account, and go to IAM roles. Search for WorkflowLoopRunner. Go to the role `dev-va-sw-WorkflowLoopRunner` You should have a role called `assume-role` with the policy 
```
{
    "Statement": {
        "Action": "sts:AssumeRole",
        "Resource": "*",
        "Effect": "Allow"
    }
}
```

**Comment by zaro0508** [comment]
> 1. To confirm, what configuration did you set for the EC2-Linux? 

I have attached my configs here: https://bin.jvnv.net/file/6QbK7/service-workbench-on-aws-issue-302.pdf

> 1. Just to confirm, did you set up a member account and assigned an index and project to that account? 

Yes, view the image in pdf doc

> 2. Let's double check that your workflowLoopRunner IAM role has the correct permission. Go to your parent AWS account, and go to IAM roles. Search for WorkflowLoopRunner. Go to the role `dev-va-sw-WorkflowLoopRunner` You should have a role called `assume-role` with the policy

Yes, i have that role and the policy matches what you expected.  view image in pdf doc

**Comment by nguyen102** [comment]
The SWB Main account need permission to access your member account. It looks like in the AWS Account section you might have added the wrong role ARN. See screenshot attached for what the role should look like.
<img width="591" alt="aws-account" src="https://user-images.githubusercontent.com/3661906/108268561-b3972100-713a-11eb-9dfe-a5cb2bcd5833.png">

The role arn should be a `cross-account-role`

When creating the member account, did you add a preexisting account or did you use the Create New Account button from within the UI?

Try adding a new member account. The easiest way to add a new account is to use the "Create New Account" button from the "Accounts" page, "AWS Accounts" Tab. 

You can find the instructions here.
https://github.com/awslabs/service-workbench-on-aws/blob/master/docs/docs/deployment/post_deployment/aws_accounts.md#creating-a-new-account

^ This page is best viewed by using docusaurus.

**Comment by zaro0508** [comment]
Ahh, I see now.  I just saw the docusaurus docs for the first time and those are much more complete than the README or the AWS SC workbench docs.  I am setting up the workbench in an AWS organizations account and I "added" accounts (as opposed to "create" accounts) from the workbench.  Reading the docs I've realized that I missed some bootstrapping steps with this scenario .   Thanks @nguyen102, I'll follow the docusaurus instructions to see if i get better results.
